src/rpc/blockchain.cpp

```diff
@@ -966,6 +967,16 @@ UniValue mempoolInfoToJSON()
     ret.push_back(Pair("size", (int64_t) mempool.size()));
     ret.push_back(Pair("bytes", (int64_t) mempool.GetTotalTxSize()));
     ret.push_back(Pair("usage", (int64_t) mempool.DynamicMemoryUsage()));
+    size_t maxmempool = GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000;
+    ret.push_back(Pair("maxmempool", (int64_t) maxmempool));
+    try
+    {
+        ret.push_back(Pair("tps", boost::lexical_cast<double>(strprintf("%.2f", mempool.TransactionsPerSecond()))));
```

Comment:
Nit: Replace with `std::floor(mempool.TransactionsPerSecond() * 100)/100` to avoid double -> string -> double conversion, boost include and try/catch.

Reply:
Hehe, nice.